docs/modules/configuring/con-configuring-vc-listeners.adoc

```diff
@@ -0,0 +1,276 @@
+  Clients make connections to the different _port numbers_ as they need to converse with each broker.
+* *SNI Host Identifies Node* - the listener assigns a distinct hostname to each broker.
+  Clients make connections to the different _hostnames_ as they need to converse with each broker.
+  The listener uses SNI (Server Name Identification) to work out which broker the client wishes to talk to
+
+IMPORTANT: You must make sure that the listener's bootstrap address and generated broker addresses are resolvable
+           and routable by the Kafka Client.  You must also make sure firewall rules permit traffic to required port numbers.
+
+== Port Identifies Node
+
+In the Port Identifies Node scheme, the virtual cluster opens a separate port for each proxied broker in addition to
+a separate port for the bootstrap.
+
+By default, this scheme assumes that the target cluster comprises three nodes with broker ids 0..2.  If this is
+inadequate, additional configuration can be provided describing the broker topology of the target broker.
+
+This scheme can be used with both plain and TLS downstream connections.
+
+This scheme works best with straightforward configurations where the target cluster uses a known minimum broker ID and
+uses stable sets of broker IDs.  For more complex cases, it is recommended to use the Server Name Identification (SNI)
+Identifies Node scheme described in the next section.
+
+IMPORTANT: When using this scheme, you have the responsibility to avoid port number collision.  In other words, ensure
+that each listener has its own range of port numbers and these do not overlap with the range used by another listener, or
+the listener of another virtual cluster.
+
+[id='con-configuring-vc-listeners-port-identifies-node-{context}']
+.Example port identifies node configuration
+
+[source,yaml]
+----
+# ...
+virtualClusters:
+  my-cluster-proxy:
+    # ...
+    listener:
+    - name: mylistener
+      portIdentifiesNode:
+        bootstrapAddress: localhost:9192 # <1>
+    # ...
+# ...
+----
+<1> The bootstrap address used by Kafka clients.
+
+With the example configuration above, the listener exposes a target cluster of up to three brokers with
+node ids `0`, `1`, `2`.   The advertised address is defaulted to that of the bootstrap host name.  Port
+numbers are assigned sequentially beginning at bootstrap port number + 1.
+
+The listener exposes the following three broker addresses:
+
+|===
+|Node Id|Broker Address
+
+|0
+|localhost:9193
+
+|1
+|localhost:9194
+
+|2
+|localhost:9195
+|===
+
+.Example port identifies node configuration with customized advertised broker address and port
+
+[source,yaml]
+----
+# ...
+virtualClusters:
+  my-cluster-proxy:
+    # ...
+    listener:
+    - name: mylistener
+      portIdentifiesNode:
+        bootstrapAddress: localhost:9192 # <1>
+        advertisedBrokerAddressPattern: mycluster.kafka.com # <2>
```

Comment:
IIRC `example.com` is reserved for exactly this sort of usage. While its not actively used `kafka.com` is a registered domain...